<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CostManagementClass</class>
 <widget class="QMainWindow" name="CostManagementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CostManagementClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>120</height>
     </size>
    </property>
    <property name="text">
     <string>新增</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFolder">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>630</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>120</height>
     </size>
    </property>
    <property name="text">
     <string>打开文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>120</height>
     </size>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinimize">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/CostManagement/resource/CostManagement.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="appTitle">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>24</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>花销管理</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="costTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QDoubleSpinBox" name="newCostInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="newCostDescription">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>560</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="curRemain">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weekCost">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="monthCost">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>600</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weekEat">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="monthEat">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>640</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="monthOther">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>640</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.989999999990687</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="curRemainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余金额：</string>
    </property>
   </widget>
   <widget class="QLabel" name="newCostLable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>金额：</string>
    </property>
   </widget>
   <widget class="QLabel" name="newCostDescriptionLable">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>570</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>描述：</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekCostLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本周总花销：</string>
    </property>
   </widget>
   <widget class="QLabel" name="monthCostLabel">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>610</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本月总花销：</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekEatLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>650</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本周饮食：</string>
    </property>
   </widget>
   <widget class="QLabel" name="monthEatLabel">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>640</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本月饮食：</string>
    </property>
   </widget>
   <widget class="QLabel" name="monthOtherLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>650</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>本月其他：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>811</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string>sdfad</string>
    </property>
   </widget>
   <zorder>labelBackground</zorder>
   <zorder>buttonAdd</zorder>
   <zorder>buttonFolder</zorder>
   <zorder>buttonDelete</zorder>
   <zorder>buttonExit</zorder>
   <zorder>buttonMinimize</zorder>
   <zorder>label</zorder>
   <zorder>appTitle</zorder>
   <zorder>costTable</zorder>
   <zorder>newCostInput</zorder>
   <zorder>newCostDescription</zorder>
   <zorder>curRemain</zorder>
   <zorder>weekCost</zorder>
   <zorder>monthCost</zorder>
   <zorder>weekEat</zorder>
   <zorder>monthEat</zorder>
   <zorder>monthOther</zorder>
   <zorder>curRemainLabel</zorder>
   <zorder>newCostLable</zorder>
   <zorder>newCostDescriptionLable</zorder>
   <zorder>weekCostLabel</zorder>
   <zorder>monthCostLabel</zorder>
   <zorder>weekEatLabel</zorder>
   <zorder>monthEatLabel</zorder>
   <zorder>monthOtherLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CostManagement.qrc"/>
 </resources>
 <connections/>
</ui>
